editor/src/core/shared/runtime-report-logs.tsx

```diff
@@ -154,3 +154,62 @@ export function useErrorOverlayRecords(): ErrorOverlayRecords {
 
   return { errorRecords, overlayErrors }
 }
+
+function filterOldPasses(errorMessages: Array<ErrorMessage>): Array<ErrorMessage> {
```

Comment:
I've moved this function (without making any changes to it) as it was causing a cyclic dependency in the tests